public/app/features/explore/utils/links.test.ts

```diff
@@ -124,8 +122,39 @@ describe('getFieldLinksForExplore', () => {
     const links = getFieldLinksForExplore({ field, rowIndex: 0, range });
     expect(links).toHaveLength(0);
   });
+
+  it('returns internal links when target contains defined template variables', () => {
```

Comment:
Can a comment explain what rowIndex points to here, such that the setup for these are the same but rowIndex 1 is a test for one condition and 0 is another?

Reply:
Good point. I've changed the code to use more explicit constant names in 1d997cf80fc2aa02a67e8e1efe87e53d160f141c. I hope it's more clear now.